enhancement_tracking/views.py

```
@@ -168,210 +174,219 @@ def home(request):
-                page += 1
-            else:
-                break
-    
-        git_tickets = git_open_tickets + git_closed_tickets
-        working['git'] = True
-    except:
-        git_tickets = []
-        working['git'] = False
-
-    try:  # Zendesk API calls to get tickets
-        zen_name_tk = profile.zen_name + '/token' # Zendesk user email set up 
-                                                  # for API token authorization.
-        # Get Zendesk tickets
-        zen_tickets = []
-        page = 1
-        while True:
-            request_zen_tickets = requests.get(BASE_ZEN_URL % profile.zen_url, 
-                                params={'query': SEARCH_QUERY % zen_limit_str,
+            request_zen_tickets = requests.get(ZEN_SEARCH_URL % \
+                                               profile.zen_url, 
+                                params={'query': ZEN_TICKET_SEARCH_QUERY,
                                         'sort_by': 'updated_at',
                                         'sort_order': 'desc',
                                         'per_page': 100,
                                         'page': page},
                                 auth=(zen_name_tk, profile.zen_token))
-            if 'error' in request_zen_tickets.json:
-                raise Exception('Error in accessing Zendesk API - %s: %s' %
-                                (request_zen_tickets.json['error'],
-                                 request_zen_tickets.json['description']))
+            if request_zen_tickets.status_code != 200:
+                raise Exception('Error in accessing Zendesk API - %s' % \
+                                request_zen_tickets.json['error'])
             zen_tickets.extend(request_zen_tickets.json['results'])
             if request_zen_tickets.json['next_page'] is not None:
                 page += 1
             else:
                 break
         
-        working['zen'] = True
+        zen_ticket_status = True
     except:
```

You almost never want to use a bare except. At least do `except Exception` so that you don't catch KeyboardInterrupt's. Also, I don't quite get the purpose of returning a tuple here. Why not just re-raise a specific Exception and then handle that Exception in the view?